### PR TITLE
New branch

### PR DESCRIPTION
**Pull request by Mehran-Banbersta**
working on timer

### CODE REVIEW

File: src/pages/welcomePage.js [hunk 1]
```diff
@@ -14,6 +14,12 @@ export const initWelcomePage = () => {
     .addEventListener('click', startQuiz);
 };
 
+
+const headerEl = document.querySelector('.header');
+
+console.log(headerEl);
```
Comment (mnvacym) [inline src/pages/welcomePage.js:19]: do you need this console.log here?

**Review by mnvacym — APPROVED** [review]

**Comment by mnvacym** [comment]
You need to run prettier

File: src/pages/welcomePage.js (hunk 1 above)
Comment (Mehran-Banbersta) [reply src/pages/welcomePage.js:19]: No, Sorry

File: package.json [hunk 2]
```diff
@@ -2,8 +2,9 @@
   "name": "js-quiz-starter",
   "version": "1.0.0",
   "scripts": {
-    "prettier:write": "prettier --write .",
-    "prettier:check": "prettier --check ."
+    "prettier": "prettier --check .",
+    "prettier:check": "npm run prettier",
+    "prettier:fix": "prettier --write ."
```
Comment (mnvacym) [inline package.json]: you don't need to change these commands

**Review by mnvacym — CHANGES_REQUESTED** [review]

**Review by mnvacym — APPROVED** [review]

**Pull request merged**